tests/test_parsl_flux_mpi_hello.py

```diff
@@ -187,3 +188,22 @@ def test_run_mpi_pi(load_config):
                 pi2_hosts.append(line.split()[1])
     # Verify each pi test ran on a different set of nodes
     assert set(pi1_hosts).intersection(pi2_hosts) == set()
+
+    with open("parsl_flux_mpi_pi1_run.out", "r") as pi1: 
+        for line in pi1:
+            if re.match(r"Start Time ", line):
+                line = line.strip().lstrip("Start Time = ")
+                pi1_start_time = dt.strptime(line, "%d/%m/%Y %H:%M:%S")
+            if re.match(r"End Time ", line):
+                line = line.strip().lstrip("End Time = ")
+                pi1_end_time = dt.strptime(line, "%d/%m/%Y %H:%M:%S")
+
+    with open("parsl_flux_mpi_pi2_run.out", "r") as pi2: 
+        for line in pi2:
+            if re.match(r"Start Time ", line):
+                line = line.strip().lstrip("Start Time = ")
+                pi2_start_time = dt.strptime(line, "%d/%m/%Y %H:%M:%S")
+            if re.match(r"End Time ", line):
+                line = line.strip().lstrip("End Time = ")
+                pi2_end_time = dt.strptime(line, "%d/%m/%Y %H:%M:%S")
+    assert pi1_start_time < pi2_start_time and pi2_start_time < pi1_end_time
```

Comment:
I was thinking there could be a better way to DRY this out since it is the same thing, but using each `*.out` file.